extension/content_script.js

```diff
@@ -411,6 +411,23 @@ function adjustSpeed(speedDelta) {
 	timeTrackerView.updateSpeed(speed,sentence_id);
 }
 
+/* Change reading speed display. This has two types:
+	1. Update - Show WPM display when users change their speed setting
+	2. Toggle - Toggle WPM display when auto-mode is turned ON/OFF
+
+	Params: 
+	- type (str)
+*/
+function changeReadingSpeedDisplay(type) {
```

Comment:
1. Maybe just split this into different fcns: update() toggle(), because i dont see any code reuse here.
2. Under toggle, if wonder if there could be a race condition where wpmDisplay visibility gets out of sync from whether or not timer is on. Might be more robust to do wpmDisplay.setAutoplayState(timer !== null)
-- update() for instance, has stop(), which might stop fadeToggle partway?

Reply:
Good point, originally when I was thinking I was going to need the animationEnum there was overlap, but now not much; I'll probably move back in-line to the keylisteners, since one-liners